app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -29,8 +40,8 @@ const StatusUpdateSection = withContext(({ store }) => {
         substatus={recentStatusUpdate.substatus}
         comment={recentStatusUpdate.comment}
         date={recentStatusUpdate.date}
-        onStatusDropdownChange={openUpdateStatusModal}
-        onAddCommentClick={openAddStatusCommentModal}
+        onStatusDropdownChange={() => onStatusDropdownChange(form)}
```

Comment:
Since these are referencing the `form` from the props we could just as easily still use them without the additional arrow function here and call to the `isFormInvalid` prop:
```js
const onStatusDropdownChange = () => !isFormInvalid ? openUpdateStatusModal() : null

onStatusDropdownChange={onStatusDropdownChange}
```

Reply:
And same for the comment